packages/discovery-provider/src/models/playlists/album_price_history.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import BigInteger, Column, DateTime, Integer, text
+from sqlalchemy.dialects.postgresql import JSONB
+
+from src.models.base import Base
+from src.models.model_utils import RepresentableMixin
+
+
+class AlbumPriceHistory(Base, RepresentableMixin):
+    __tablename__ = "album_price_history"
+
+    playlist_id = Column(Integer, nullable=False, primary_key=True)
+    splits = Column(JSONB(), nullable=False)
```

Comment:
I wonder if it makes sense for us to add doc comments about the exact structure of said JSONB?